README.md

```diff
@@ -0,0 +1,29 @@
+# Discord
+
+Discord is a free all-in-one messaging, voice and video client that's available on your computer and phone.
+
+This repo hosts the flatpak wrapper for [Discord](https://discord.com/), available at [Flathub](https://flathub.org/apps/details/com.discordapp.Discord).
+
+```sh
+flatpak install flathub com.discordapp.Discord
+flatpak run com.discordapp.Discord
+```
+
+
+## Differences in flatpak version
+
+The flatpak version runs in a sandbox to provide better safety and privacy for users.
+
+However, this sandboxing prevents the following features from working:
+
+- **Game Activity**: This flatpak version of Discord cannot scan running processes to detect running games.  
+  There is currently no workaround or solution for this limitation.
+- **Drag and Drop Files**: To work around this now, you can change sandbox permissions of installed flatpak applications to give Discord file system access (for example, with [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) or with `flatpak override --user --filesystem=home:ro com.discordapp.Discord`).
```

Comment:
```suggestion
- **Drag and Drop Files**: To work around this now, you can change sandbox permissions of installed flatpak applications to give Discord file system access:

```sh
flatpak override --filesystem=home:ro com.discordapp.Discord```
```

---

At line `8` you typed `flatpak install flathub com.discordapp.Discord` (without the `--user` flag), so when needing to change the permissions, you can't use `--user` since you installed it in your system.

IMO while I'm a big fan of Flatseal, I don't think instructions with Flatseal should be included in READMEs, or at least make it look better by doing a step-by-step guide rather than tell the user they can use Flatseal with no further instructions. It clutters a lot and makes it hard to follow through the README.

Reply:
> you can't use --user since you installed it in your system.

@TheEvilSkeleton I just tested this on my system installation of discord flatpak and yes it's working with --user override. See my comments above. https://github.com/flathub/com.discordapp.Discord/pull/133#discussion_r552984244

Reply:
Without --user, I need sudo to override

Reply:
Wtf? I just tested it in a VM and it worked. Forget what I've said :P

Reply:
flatpak docs is definetly confusing

Reply:
I updated the suggestion so it should look better.

Reply:
> flatpak docs is definetly confusing

Agreed

Reply:
"allow edits by mantainers" is on, so you guys can commit whatever you want too, I think

Reply:
I'm not a maintainer, just a user lurking around.

Reply:
Ah, ok. I'm a noob doing this, so I don't know how this all works

Reply:
Pretty sure you can press on "Accept changes", or something like that right?

Reply:
yeah, I'll add that with the other link thing
Do you know if I can just add <a href=... on the appdata.xml?

Reply:
No idea.